### ISSUE TITLE
Option to disable scrobbling video chapters on YouTube

### ISSUE THREAD

**Issue by huuishuu** [web-scrobbler/web-scrobbler#3015]
**Is your feature request related to a problem? Please describe.**
YouTube has recently started enabling auto-chapters on a lot of videos, which often includes random song lyrics on music videos, leading to broken scrobbles.

**Describe the solution you'd like**
An option in the settings for Web Scrobbler to not scrobble video chapters and just scrobble the full video instead. It would ruin listening to album videos, sure, but it'd be better than having it scrobble random song lyrics that YouTube thought was a good idea to do or random podcast segments and just the full podcast instead.

**Describe alternatives you've considered**
An option to set it "per video" would be okay too, retaining compatibility for scrobbling albums.

**Additional context**
![](https://i.imgur.com/UL8YFWj.png)

![](https://i.imgur.com/Ix7i7RO.png)

^ these perfectly illustrate why this is insanely annoying.

**Comment by dpraul** [comment]
I get bitten by this same issue with the auto-generated chapters a lot - scrobbling "Chapter 1", "Chapter 2", etc.

I wonder if a good solution here would be to, as a rule, just not scrobble chapters when they're auto-generated. The only time I've found chapters useful for scrobbling is when they're manually included, since then they'll have usually have track information. The auto-generated ones never do.

**Comment by rob9315** [comment]
The auto-chapters being used for scrobble information should be fixed when #5873 gets merged.

Is disabling normal chapters being used for scrobbling still a concern?
if not, I'll mark #5873 as resolving this issue

**Comment by Californ1a** [comment]
> Is disabling normal chapters being used for scrobbling still a concern?

Yes, there are still some cases where it would be a good option to be able to disable manual chapter scrobbling, and scrobble the full video instead on a per-video basis, such as this: https://www.youtube.com/watch?v=y0ZoX4dBvwk
